components/new-member-list-item/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from './new-member-list-item.module.scss';
+import { motion } from 'framer-motion';
+
+const NewMemberListItem = ({ newMemberDetails }) => {
+  const { username } = newMemberDetails;
+  const fullName = `${newMemberDetails.first_name + ' ' + newMemberDetails.last_name}`;
+
+  const brokenImageHandler = (e) => {
+    e.target.src = '/images/Avatar.png';
+  };
+
+  return (
+    <div className={classNames.container}>
+      <motion.img
+        layoutId={username}
+        src={newMemberDetails.img_url + `?${Math.random() * 100}`}
+        onError={brokenImageHandler}
+        className={classNames.imgContainer}
+        alt={username}
+      />
+      <h2 className={classNames.nameOfPerson}>
+        {fullName.length > 1
+          ? `${newMemberDetails['first_name']} ${newMemberDetails['last_name']}`
+          : username}
+      </h2>
+    </div>
+  );
+};
+
+NewMemberListItem.propTypes = {
+  newMemberDetails: PropTypes.object
```

Comment:
Since we know the shape of how our `newMemberDetails` object looks like, can we use `PropTypes.shapeOf()`?

Reply:
should it be like this?
![image](https://user-images.githubusercontent.com/45538392/118701494-673ea880-b831-11eb-94b9-bde11841594c.png)